pipelines/shared/Jenkinsfile

```diff
@@ -77,7 +77,7 @@ pipeline {
                 CONFIG_ROLE_ID = credentials('knox-jenkins-jenkins-apps-prod-role-id')
                 NR_BROKER_JWT = credentials('nr-broker-jwt')
                 AUTHFILE = "auth.json"
-                CAUSE_USER_ID = "${params.userId != "null" ? "${params.userId}" : "github@internal"}"
+                CAUSE_USER_ID = "${params.userId != null && params.userId != "" ? "${params.userId}" : "github@internal"}"
```

Comment:
Enclosing "" might not be necessary. Tried?

CAUSE_USER_ID = params.userId != null && params.userId != "" ? "params.userId : "github@internal"